changelogs/fragments/3634-pipx-improve-changed.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
In the PR you are saying it's a bugfix. Can you please elaborate whether this is a bugfix (if it is, please change this to `bugfixes:`) or not?

Reply:
It was late last night when I submitted, guess I didn't think it through. Just changed to Feature PR instead.

Reply:
It is not a bug. It is not a feature either, it is a minor improvement on the code, with no visible change to modules that use it.